src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -50,36 +50,44 @@
     @InjectMocks
     private RoomService service;
 
+    @Mock
+    private static Utility utility;
```

Comment:
If you do decide to go the autowired route, I could be wrong but I think this would need to be above `service` and be non-static

Reply:
Also just a realization, if you decide to go with the UUID route, then there may not actually be a need to extract the generateRoomId method into Utility at all (so that might eliminate some of the difficulties with mocking)

Reply:
Yeah I realized this too, I could potentially just put that back in the `Room` class -- we can talk about this today during our standup.

Reply:
Done! I don't think the `static` component mattered, but I removed it anyway.